### PR TITLE
TEDP3 + cdt 3.0.1 upgrade + general refactor

### LINKED ISSUE #5: TEDP3 changes
Add two actions to set payments for two new accounts: `ignitegrants`, `tlosfuelfund`,

max amount for fuel fund: `55890` per day
max amount for ignite grants: `16438` per day

### PR DESCRIPTION
**Pull request by guilledk**
Closes #5 

## Description

Add TEDP3 functions `setignite` and `setfuel`
Drop most contracts as they are not needed in the tests, also drop system tests
Refactored a bunch of constant duplication to a new file, which gets imported on both the test suite and the contract itself Refactored error logging on `eosio.tedp` setpayout assertions
Dropped old `ACTION` 1.6.3 standard annotations
Refactored some long lines of code to make them readable on 80 char terminals
Modified CMake file structure
Add testing helpers for new ignite and fuel machinery
Increase readability of tests
Fix broken `pay_flow` test

## Test scenarios

Using this cdt 3.0.1 docker file: https://github.com/guilledk/telos-cdt-docker/blob/master/Dockerfile.cdt-3.0.1
Using this command ran from the root of the repo: 
`docker run -it --rm --mount type=bind,source=$(pwd),target=/root/target cdt:3.0.1 bash -c "cd build && cmake .. && make && cd tests && ./unit_test"`

Should print `*** No errors detected` after compilation and test execution. 

## Checklist:

-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have cleaned up the code in the areas my change touches
-   [x] My changes generate no new warnings
-   [x] Any dependent changes have been merged and published in downstream modules
-   [x] I have checked my code and corrected any misspellings
-   [x] I have removed any unnecessary console messages

### CODE REVIEW

File: src/include/eosio.tedp/tedp.constants.hpp [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+// const uint64_t max_drawdown_amount = 173000;
+
+// 60sec * 60min * 24hrs
+const uint64_t daily_interval = 86400;
+
+// 60sec * 30min
+const uint64_t rex_interval = 1800;
+
+// 500k * 12 months / 365 days = 16438.3561
+const uint64_t max_econdev_amount = 16438;
```
Comment (poplexity) [inline src/include/eosio.tedp/tedp.constants.hpp]: econdevfund is 0 now

File: src/include/eosio.tedp/tedp.constants.hpp [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+// 1mil * 12 months / 365 days = 32876.7123
+const uint64_t max_tf_amount = 32876;
+
+// 300k * 12 months / 365 days = 9863.0136
+const uint64_t max_coredev_amount = 9863;
```
Comment (poplexity) [inline src/include/eosio.tedp/tedp.constants.hpp]: 13150 for TCD, 400k/mo

File: src/include/eosio.tedp/tedp.constants.hpp [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+// 1.7mil * 12 months / 365 days / 24hrs / 2 (every half hour) = 1164.3835
+const uint64_t max_rex_amount = 1165;
+
+const uint64_t max_ignitegrants_amount = 16438;
+
+const uint64_t max_tlosfuel_amount = 55890;
```
Comment (poplexity) [inline src/include/eosio.tedp/tedp.constants.hpp:31]: 1.7mil for fuel a month, to add as comment formula above

File: src/include/eosio.tedp/tedp.constants.hpp [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+// 1mil * 12 months / 365 days = 32876.7123
+const uint64_t max_tf_amount = 32876;
+
+// 300k * 12 months / 365 days = 9863.0136
+const uint64_t max_coredev_amount = 9863;
+
+// 1.7mil * 12 months / 365 days / 24hrs / 2 (every half hour) = 1164.3835
+const uint64_t max_rex_amount = 1165;
+
+const uint64_t max_ignitegrants_amount = 16438;
```
Comment (poplexity) [inline src/include/eosio.tedp/tedp.constants.hpp:28]: 500k/mo for ignite a month, to add as comment formula above

**Comment by guilledk** [comment]
Deployed to testnet on tx `5ea2f84f8499c55d8b8ace151ec79381bebfbc8aaf5616c2db84008f6b52ebdf`

**Pull request merged**